draft-ietf-quic-recovery.md

```diff
@@ -1407,15 +1409,23 @@ Invoked when an ACK frame with an ECN section is received from the peer.
 Invoked from DetectLostPackets when packets are deemed lost.
 
 ~~~
-   InPersistentCongestion(largest_lost_packet):
+   InPersistentCongestion(lost_packets):
      pto = smoothed_rtt + max(4 * rttvar, kGranularity) +
        max_ack_delay
      congestion_period = pto * kPersistentCongestionThreshold
+
+     earliest_lost_packet = lost_packets.first()
+     largest_lost_packet = lost_packets.last()
+
      // Determine if all packets in the time period before the
      // newest lost packet, including the edges, are marked
      // lost
-     return AreAllPacketsLost(largest_lost_packet,
-                              congestion_period)
+     all_packets_lost =
+       lost_packets.length() == (lost_packets.last().packet_number -
+                                 lost_packets.first().packet_number - 1)
```

Comment:
My mistake on the example, this should be:

```suggestion
                                 lost_packets.first().packet_number + 1)
```

Reply:
Fixed.